src/bootstrap/src/core/builder.rs

```diff
@@ -1667,21 +1669,20 @@ impl<'a> Builder<'a> {
             }
         }
 
-        if let Some(host_linker) = self.linker(compiler.host) {
-            hostflags.arg(format!("-Clinker={}", host_linker.display()));
-        }
-        if self.is_fuse_ld_lld(compiler.host) {
-            hostflags.arg("-Clink-args=-fuse-ld=lld");
-        }
+        linker_args(self, compiler.host, LldThreads::Yes).into_iter().for_each(|flag| {
```

Comment:
```suggestion
        for arg in linker_args(self, compiler.host, LldThreads::Yes) {
```
I haven't seen cases in which `for_each` would look better than a loop (maybe only in long iterator chains).